week1/HomeworkWeek1.py

```diff
@@ -0,0 +1,77 @@
+# Task 1 - Write a python script to print your name and age
+
+name = input("Enter your name")
+age = int(input("Enter your age"))
+
+print("Your name is : ", name, " Your age is : ", age)
+
+# Task 2 - Create a list of your 5 favorite movies and store it in the variable
+
+movies = []
+for i in range(5):
+    movie = input(" Enter your favorite movies")
+    movies.append(movie)
+
+print("Your 5 Fav movies are : ", movies)
+
+# Task 3 - Write a Python program to display the first and last colors from the following list.
+
+color_list = ["Red", "Green", "White", "Black"]
+print(color_list[0], color_list[-1])
+
+# Task 4 - Write a Python script to add a key to a dictionary
+numbers = {0: 10, 1: 20}
+numbers[2] = 30
+print(numbers)
+
+# Task 5 - Write a Python program to calculate body mass index.
+name = input("enter your name")
+weight = float(input("enter your weight in kg"))
+height = float(input("enter your height in meter"))
+
+bmi = weight/height*height
+print(name, "'s BMI is ", bmi)
+
+# Task 6 - Guess a number game - between 1 to 9.
+
+guessnumber = 7
+while True:
+    userguess = int(input("Enter your guess number"))
+    if userguess == guessnumber:
+        print("Well guessed!")
+        break
+
+# Task 7 - Create a tuple with different data types
+
+tuple1 = ("hello world", 134, True, 40.4)
```

Comment:
It's a silly example exercise, but using a variable name like `tuple1` which is very close to `tuple` (a word from pythons syntax) is not a good choice. What about `example_tuple` or `sample_tuple`.

Reply:
done.